ewc-rfc005-issue-legal-person-identification-data.md

```diff
@@ -31,34 +31,42 @@ The LPID issuance process is implemented by PID providers according to national
 
 4. **Status:** The status of the legal person must be verified. These states are not yet regulated, but at the very least, the legal person for which an LPID is requested must be registered at a business register.
 
-The issuance process can either start in:
-1. a wallet application, the grahical user interface of the wallet *or*
+The issuance process can be done in:
```

Comment:
PID issuance can only be made, or be done, by a PID provider. The initiation of the process can be one of the starting points described.